### DISCUSSION TITLE
[Application Development] <Tpitle>

### DISCUSSION THREAD

**Discussion by Pazibad** [ethereum-optimism/developers#311] [Feedback]
### Feedback

https://www.bitgetapp.com/support/articles/12560603775388

### Additional Information

blockchains/ethereum/assets/0x576e2BeD8F7b46D34016198911Cdf9886f78bea7/info.json

**Comment by sbvegan** [comment]
Hey can you provide more context here? We can reopen this if you have a direct ask or feedback to share.
